docs/specs/servers/blockchain/blockchain-bundler-api.md

```diff
@@ -0,0 +1,48 @@
+# Blockchain API Bundlers interaction
+
+This API provides a bundlers JSON-RPC interaction. 
+The supported `userOperation` interaction schema is according to v0.7 ERC-4337 specification.
+
+## Make a bundler JSON-RPC request
+
+Used to make a JSON-RPC call to the designated bundler.
+
+`POST /v1/bundler?projectId={projectId}&chainId={chainId}&bundler={bundlerName}`
+
+* `projectId` - Required. The project identifier.
+* `chainId` - CAIP-2 chain identifier.
+* `bundlerName` - (Optional) Bundler name to use.
+
+The list of currently supported bundler names: 
+* `pimlico` - [Pimlico bundler](https://docs.pimlico.io/permissionless/reference/bundler-actions/sendUserOperation).
+
+#### Success response body:
+
+```typescript
+{
+    id:  number, // Unique identifier. Default is 1.
+    jsonrpc: string, // JSON-RPC version. Default 2.0
+    method: string, // ERC-4337 bundler operation methods. Please check supported methods below.
+    params: string[], // Array of string variables input for the method.
+}
+```
+
+List of supported `method` values:
+* `wallet_getCallsStatus` - Returns the status of a call batch that was sent via `wallet_sendCalls`.
+* `wallet_showCallsStatus` - Requests that a wallet shows information about a given call bundle that was sent with `wallet_sendCalls`.
```

Comment:
in the current demo PoC, we're fetching `eth_getUserOperationReceipt` are we going to support it?

Reply:
Updated by adding `eth_getUserOperationReceipt`.

Reply:
I can add this operation also if needed. Let's support it for now.